packages/amazon-cognito-identity-js/src/UserAgent.js

```diff
@@ -24,3 +25,12 @@ export const appendToCognitoUserAgent = content => {
 
 // class for defining the amzn user-agent
 export default UserAgent;
+
+export const getAmplifyUserAgentString = ({ action, framework } = {}) => {
+	rerunFrameworkDetection();
+	const userAgent = `${UserAgent.prototype.userAgent} ${category}/${
+		action ? action : AuthAction.None
+	} framework/${framework ? framework : Platform.framework}`;
```

Comment:
Nit, but the linter makes this a little tricky to read. Maybe something like:

```suggestion
    const uaAction = action ?? AuthAction.None;
    const uaFramework = framework ?? Platform.framework;
	const userAgent = `${UserAgent.prototype.userAgent} ${category}/${uaAction} framework/${uaFramework}`;
```

Reply:
Good point, it is a little ugly